src/sandstorm/grain.capnp

```diff
@@ -467,6 +588,40 @@ interface SessionContext {
   #     // pass event.data.token to your app's server and call SandstormApi.restore() with it
   #   }
   # }, false)
+
+  provide @4 (cap :Capability, requiredPermissions :PermissionSet,
+              descriptor :PowerboxDescriptor, displayInfo :PowerboxDisplayInfo);
+  # For sessions started with `newRequestSession()`, fulfills the original request. If only one
+  # capability was requested, the powerbox will close upon `provide()` being called. If multiple
+  # capabilities were requested, then the powerbox remains open and `provide()` may be called
+  # repeatedly.
+  #
+  # If the session was not started with `newRequestSession()`, this method is equivalent to
+  # `offer()`. This can be helpful when building a UI that can be used both embedded in the
+  # powerbox and stand-alone.
```

Comment:
Is there any situation where we would want `offer()` and `provide()` to act differently? Do they need to be distinct methods?